README.md

```diff
@@ -11,8 +11,13 @@ The t & e come for free.
 
 ## Build
 
-In the root directory of the project run `wasm-pack build`, this should produce a `pkg` folder.
-Next in the `www` direcotry run `npm run build`, this should produce a `dist` folder.
-The `dist` folder is a self-contained website with html, wasm modules, and javascript.
+In the root directory of the project run `wasm-pack build`, this should produce a `pkg` folder. Next in the `www` directory install the following modules by running the following.
 
-Alternatively, `npm start` in the `www` directory will start a local server for the frontend.
+* `npm install webpack`
+* `npm install webpack-cli`
```

Comment:
Wow really nice

Reply:
since there's a already a `package.json` file listing the required dependencies, shouldn't a single `npm install` be enough here ?

Reply:
Hey, yes probably.

However, Macy had some difficulties getting set up so I asked her to document things a little more succinctly.

Also sorry if that comment confused you it was mostly meant as a joke while I was showing Macy how to Fork and Pull Request!